src/Fantomas/CodePrinter.fs

```diff
@@ -2000,7 +2000,7 @@ and genExpr astContext synExpr =
                 mkRange "IN" binding.RangeOfBindingAndRhs.End e.Range.Start
 
             Map.tryFindOrEmptyList IN ctx.TriviaTokenNodes
-            |> TriviaHelpers.``keyword token inside range`` inRange
+            |> TriviaHelpers.``keyword token after start column and on same line`` inRange
```

Comment:
Probably a better thing to do here would be looking inside the range between the last binding end and the body expression start.

Reply:
Yeah, thought of that but we capture nested `LetOrUse` expressions so that information is lost in CodePrinter.
See https://github.com/fsprojects/fantomas/blob/3035e5aa6f2ce96241e31eee04edd9ef73986371/src/Fantomas/SourceParser.fs#L876